tests/projects/c++/modules/circular_dependency/src/hello.mpp

```diff
@@ -0,0 +1,13 @@
+export module hello;
+
+import hello3;
+
+export namespace hello {
+    class say {
+    public:
+        say(int data);
+        void hello();
```

Comment:
why contain this patch https://github.com/xmake-io/xmake/pull/4677/files?

Reply:
i did something wrong with git :D